### PR TITLE
Create an example datasource and dashboard via Makefile

### PR DESCRIPTION
**Pull request by zhuje**
- Remove `dashboard-config-map.yaml` from Helm Charts
- Create an example dashboard and datasource configMap users can deploy via Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,3 +48,7 @@ build-image:
 
 .PHONY: install
 install: install-frontend install-backend
+
+.PHONY: example
+example: 
+	cd docs && oc apply -f prometheus-datasource-example.yaml && oc apply -f prometheus-dashboard-example.yaml
```
Comment (jgbernalp) [inline Makefile:54]:
```suggestion
	cd docs && oc apply -f prometheus-datasource-example.yaml && oc apply -f prometheus-dashboard-example.yaml

```

Comment (zhuje) [reply Makefile]: Committed and squashed into the latest push.

File: README.md [hunk 2]
```diff
@@ -48,3 +48,9 @@ helm upgrade -i console-dashboards-plugin charts/console-dashboards-plugin -n co
 ## Add a new Datasource
 
 See [add datasource docs](docs/add-datasource.md)
+
+### Deploy an Example Datasource and Dashboard: 
+1. `oc login` (requires [oc](https://console.redhat.com/openshift/downloads) and an [OpenShift cluster](https://console.redhat.com/openshift/create))
+2. Deploy the plugin on the cluster `helm upgrade -i console-dashboards-plugin charts/console-dashboards-plugin -n console-dashboards --create-namespace`
+3. Run `make example`
```
[the remark this answers is not in the document]
Comment (zhuje) [reply README.md]: Committed and squashed into the latest push.

**Pull request merged**